examples/MintsMultipleTokens.js

```diff
@@ -0,0 +1,11 @@
+var rawData1 = require('Scripts/redeemer1.plutus');
```

Comment:
Please replace all `var`s with `const`. Also, the variable names here aren't descriptive